src/Fixer/Strict/DeclareStrictTypesFixer.php

```diff
@@ -161,10 +161,6 @@ private function insertSequence(Tokens $tokens)
         }
 
         $content = $tokens[1 + $endIndex]->getContent();
-        if (false !== strpos($content, "\n")) {
-            return;
-        }
-
-        $tokens[1 + $endIndex] = new Token(array(T_WHITESPACE, $lineEnding.ltrim($content)));
+        $tokens[1 + $endIndex] = new Token(array(T_WHITESPACE, $lineEnding.ltrim($content, $this->whitespacesConfig->getIndent())));
```

Comment:
Using `$this->whitespacesConfig->getIndent()` as second parameter of `ltrim()` looks weird, why not use `" \t"`?

Reply:
I don't know, `" \t"` seems to be better choice indeed.